code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -445,25 +419,39 @@
 			close_machine(target)
 
 /obj/machinery/cryo_cell/screwdriver_act(mob/living/user, obj/item/tool)
+	. = ITEM_INTERACT_BLOCKING
+	if(on)
+		balloon_alert(user, "turn off!")
+		return
+	if(occupant)
+		balloon_alert(user, "occupant inside!")
+		return
 
-	if(!on && !occupant && !state_open && (default_deconstruction_screwdriver(user, "pod-off", "pod-off", tool)))
+	if(default_deconstruction_screwdriver(user, "pod-off", "pod-off", tool))
 		update_appearance()
-	else
-		to_chat(user, "<span class='warning'>You can't access the maintenance panel while the pod is " \
-		+ (on ? "active" : (occupant ? "full" : "open")) + "!</span>")
-	return ITEM_INTERACT_SUCCESS
+		return ITEM_INTERACT_SUCCESS
 
 /obj/machinery/cryo_cell/crowbar_act(mob/living/user, obj/item/tool)
-	if(on || state_open)
-		return FALSE
-	if(!panel_open)
-		balloon_alert(user, "open panel!")
-		return ITEM_INTERACT_SUCCESS
+	. = ITEM_INTERACT_BLOCKING
+	if(on)
+		balloon_alert(user, "turn off!")
+		return
 
-	var/unsafe_wrenching = FALSE
-	var/filled_pipe = FALSE
-	var/datum/gas_mixture/environment_air = loc.return_air()
+	var/unsafe_release = FALSE
 	var/datum/gas_mixture/inside_air = internal_connector.gas_connector.airs[1]
+	if(inside_air.total_moles() > 2 * ONE_ATMOSPHERE)
+		to_chat(user, span_warning("As you begin prying \the [src] a gush of air blows in your face... maybe you should reconsider?"))
```

Comment:
Does it make sense to expel the internal gas just for prying someone out? Also there's nothing stopping this from running if the machine is already open or lacking any occupants.

Reply:
Yeah i need to add more checks to this.